rust/sample-plugin/src/main.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2016 Google Inc. All rights reserved.
+// Copyright 2018 Google LLC
```

Comment:
There's no need to update copyright notices on existing files, only to make sure they're right on new files.

Reply:
@betterclever were talking about licences and he pointed out that this file, which was added recently, has the old licence. Not sure if that matters, but this file _was_ added in 2018.